docs/ldml/tr35-general.md

```diff
@@ -2165,13 +2165,34 @@ List patterns can be used to format variable-length lists of things in a locale-
 </listPatterns>
 ```
 
-The data is used as follows: If there is a type type matches exactly the number of elements in the desired list (such as "2" in the above list), then use that pattern. Otherwise,
+Each pattern satisifies the following conditions:
+<ul>
+    <li>it contains both a <code>{0}</code> and a <code>{1}</code> placeholder</li>
+    <li>the <code>{0}</code> placeholder appears before the <code>{1}</code> placeholder</li>
+    <li>"start" and "middle" patterns end with the <code>{1}</code> placeholder</li>
+    <li>"middle" and "end" patterns begin with the <code>{0}</code> placeholder</li> 
+</ul>
```

Comment:
Add clarifying sentence, something like: 

That is, 
* all patterns can have text between the placeholders
* only the start and '2' patterns can have literal text after the placeholders, and 
* only the end and '2' patterns can have literal text before the placeholders.

Reply:
^ that's actually the wrong way around 😛